docs/ref/models/database-functions.txt

```diff
@@ -800,6 +800,27 @@ Usage example::
     >>> print(author.name_lower)
     margaret smith
 
+``LPad``
+--------
+
+.. class:: LPad(expression, length, fill_text=Value(' '), **extra)
+
+.. versionadded:: 2.1
+
+Returns the value of the given text field or expression padded on the left-side
+with a ``fill_text`` to a specified ``length``. The default ``fill_text`` is
```

Comment:
"padded on the left side with `fill_text` so that the resulting value is `length` characters long."